go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/jubnzv/go-tmux
+
+go 1.15
```

Comment:
Does this mean that this module requires at least go 1.15? I used it with go 1.12 and everything worked correctly.

Reply:
I can get this to a lower version, it was just an auto generated go mod file.

Reply:
done